### PR TITLE
ingress-controller: Fix handling of deleted certificates

### PR DESCRIPTION
**Pull request by mikkeloscar**
Ref: https://github.com/zalando-incubator/kube-ingress-aws-controller/pull/424

### CODE REVIEW

**Comment by szuecs** [comment]
👎

**Comment by szuecs** [comment]
Poll time changes need a discussion, because it was a mitigation for aws rate limits incident. You can always force a reset by deleting the pod.

File: cluster/config-defaults.yaml [hunk 1]
```diff
@@ -27,6 +27,7 @@ kube_aws_ingress_controller_nlb_enabled: "false"
 kube_aws_ingress_controller_nlb_cross_zone: "true"
 # sets the default LB type: "network" or "application" are valid choices
 kube_aws_ingress_default_lb_type: "application"
+kube_aws_ingress_controller_cert_polling_interval: "1m"
```
Comment (szuecs) [inline cluster/config-defaults.yaml]: please change default to 2m, thanks

**Comment by szuecs** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**